index.js

```diff
@@ -43,17 +38,32 @@ module.exports = (robot, options, visit) => {
     });
   }
 
+  // Trigger the event
+  function trigger(event) {
+    robot.webhook.emit(event.event, event);
+  }
+
   function schedule(installation, repository) {
```

Comment:
So I have played with the interval in the past (really short when testing, long in production), which is not really possible via the current code (would be interesting if we supported an additional `options.intervalExact` or something like that. I would assume that you would want custom intervals sooner rather than later, and once you did that, you would want a way to identify which schedule is which.

But agree, with the ability to do custom scheduling (or even have much control over it), it's not of much use.

Just my thoughts.